core/gui/src/app/workspace/component/left-panel/settings/settings.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from "@angular/core";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
+import { WorkflowActionService } from "../../../service/workflow-graph/model/workflow-action.service";
+import { WorkflowPersistService } from "../../../../common/service/workflow-persist/workflow-persist.service";
+import { UserService } from "../../../../common/service/user/user.service";
+import { NotificationService } from "src/app/common/service/notification/notification.service";
+
+@UntilDestroy()
+@Component({
+  selector: "texera-settings",
+  templateUrl: "./settings.component.html",
+  styleUrls: ["./settings.component.scss"],
+})
+export class SettingsComponent implements OnInit {
+  settingsForm!: FormGroup;
+  currentBatchSize!: number;
+  isSaving: boolean = false;
+
+  constructor(
+    private fb: FormBuilder,
+    private workflowActionService: WorkflowActionService,
+    private workflowPersistService: WorkflowPersistService,
+    private userService: UserService,
+    private notificationService: NotificationService
+  ) {}
+
+  ngOnInit(): void {
+    this.currentBatchSize = this.workflowActionService.getWorkflowContent().settings.batchSize || 400;
```

Comment:
Question: check if the frontend allows a global variable with a value of "400".  If so, use that variable instead of the constant "400".

Reply:
Check if the constant "400" can be sent from the backend to the frontend.

Reply:
I think we can place the value 400 in the environment.default.ts file. This makes it globally accessible and able for users to modify.